app/models/data_center.rb

```diff
@@ -73,6 +73,12 @@ class DataCenter < ApplicationRecord
   has_many :validator_ips_active, through: :data_center_hosts
   has_many :validators, through: :data_center_hosts
   has_many :validator_score_v1s, through: :data_center_hosts
+  has_many :gossip_nodes, through: :data_center_hosts
+  has_many :data_center_stats do
+    def by_network(network)
+      where(network: network)
```

Comment:
```suggestion
      find_by(network: network)
```
Maybe it's better since there should be only one record per network for data_center if I understand correctly